### PR TITLE
Implemented searches.video.by_tag

### PR DESCRIPTION
**Pull request by BenjaminSchaaf**
Added `searches.video.by_tag` with spec

### CODE REVIEW

**Comment by avokhmin** [comment]
@BenjaminSchaaf , can you do a `git rebase` please?

**Comment by BenjaminSchaaf** [comment]
done

File: lib/youku/v2/searches.rb [hunk 1]
```diff
@@ -12,6 +12,30 @@ def video
...
+        #
+        # tag       - The comma separated String of tags.
+        # category  - The String category.
+        # period    - The String period.
+        # orderby   - The String order of videos.
+        # page      - The Integer page number.
+        # count     - The Integer page size.
+        #
+        # Returns the instance of Youku::V2::Request.
+        def by_tag(tag: nil, category: nil, period: 'today', orderby: 'relevance', page: 1, count: 20)
```
Comment (avokhmin) [inline lib/youku/v2/searches.rb]: http://cloud.youku.com/docs?id=80

> userid    int/string  false       指定用户查找（指定用户将不支持多标签搜索，并且仅支持排序，其他参数失效）

can you add too? Or why not?

**Comment by BenjaminSchaaf** [comment]
Ah, I missed that. Updated

File: lib/youku/v2/searches.rb [hunk 2]
```diff
@@ -12,6 +12,31 @@ def video
...
+        # Public: Get videos by tag.
+        #
+        # See: http://open.youku.com/docs?id=80
+        #
+        # tag       - The comma separated String of tags.
+        # category  - The String category.
+        # period    - The String period.
+        # orderby   - The String order of videos.
+        # page      - The Integer page number.
+        # count     - The Integer page size.
```
Comment (avokhmin) [inline lib/youku/v2/searches.rb:25]: > userid - The String or Integer ...

and comments :)

**Comment by avokhmin** [comment]
@BenjaminSchaaf also, will you do a new pull-requests or I can create a new release today?

**Comment by BenjaminSchaaf** [comment]
This is the last one, feel free to make a release.

**Comment by BenjaminSchaaf** [comment]
Added docs

**Pull request merged**